src/k8s/pkg/k8sd/setup/kubelet_test.go

```diff
@@ -109,27 +109,28 @@ func TestKubelet(t *testing.T) {
 			key         string
 			expectedVal string
 		}{
+			// Overwritten by extraArgs
```

Comment:
i hate to be that guy, but please keep the extra args at the end of the list, otherwise this comment here doesn't make any sense. these extra args, can in fact be sorted again

Reply:
No worries, I did too much sorting.